modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -3,38 +3,132 @@
  * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
  * PURPOSE:         Test for PrivateExtractIcons
  * PROGRAMMER:      Hermes Belusca-Maito
+ *                  Doug Lyons <douglyons@douglyons.com>
  */
 
 #include "precomp.h"
+#include <stdio.h>
+
+BOOL FileExists(LPCSTR FileName)
+{
+    DWORD Attribute = GetFileAttributes(FileName);
+
+    return (Attribute != INVALID_FILE_ATTRIBUTES &&
+            !(Attribute & FILE_ATTRIBUTE_DIRECTORY));
+}
+
+BOOL ResourceToFile(INT i, LPCSTR FileName)
```

Comment:
You can keep using unicode file string names if you use `_wfopen`

Reply:
Applied the Unicode strategy 👍